week-2/Hathiramani - Lesson 2 H.W./index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <!--Metadata goes here-->
+  <title>Portfolio</title>
+</head>
+<body>
+  <!--Content goes here-->
+  <h1>Utsa's Portfolio</h1>
+  <p>This webpage will be a platform to showcase main projects I've worked on. </p>
+  <h2>How the webpage will be divided into three pages:</h2>
+  <ol>
+    <p><li>Absolut Craft:</li></p>
+    <ul>
+    <li>This project is transforming packaging. The aim of the project was to make a new product for the brand Absolut in the food or cosmetics industry.</li>
+    <li>Absolut Craft is an infusion kit allows the user to create their own personalised flavours.</li>
+    <li>The page for this project will show concepts, moodboards, renders and final product images and branding.</li>
+  </ul>
+    <p><li>Absolut Challenge</li></p>
```

Comment:
Normally order doesn't matter, but you should have an `<li>` element as the outermost element in this case. `<p></p>` can go inside of it.